lib/elixir/lib/io/ansi/docs.ex

```diff
@@ -291,6 +291,8 @@ defmodule IO.ANSI.Docs do
     col =
       col
       |> String.replace("\\\|", "|")
+      |> String.replace(~r/-+/, "-")
```

Comment:
@josevalim That's not quite the same thing. My version replaces `:-------` with `:-`.